src/components/UserForm/UserForm.js

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
+import EditPopup from '../Popup/EditPopup';
 import UserField from '../UserField/UserField';
-import "./userForm.css"
+import './userForm.css';
+import * as EmailValidator from 'email-validator';
```

Comment:
don't use package for validation use regular expressions, if you don't know them, there is no problem just learn them :D :D :D